scripts/requirements.mbed.txt

```diff
@@ -1,3 +1,8 @@
-mbed-tools>=7.49.1 ; platform_machine != 'aarch64' and sys_platform == 'linux'
+#mbed-tools>=7.49.1 ; platform_machine != 'aarch64' and sys_platform == 'linux'
```

Comment:
Is this line commented out on purpose?

Reply:
Yes, original PR mentined that mbed-tools is not installable.

Likely because click module restrictions: for some reason mbed-tools requires click<8, so it has a 'enforce old version of package', this is the reason it got commented out instead of others things being disabled (others require newer version of packages, not older)